src/integration-tests/test_admin_client.py

```diff
@@ -64,7 +64,7 @@ def post_n_msgs(
     for _ in range(task.num):
         res = producer.post(task.uri, payload=["msg"], wait_ack=True)
         assert Client.e_SUCCESS == res
-    producer.close(task.uri)
+    producer.close(task.uri, succeed=True)
```

Comment:
This close was nonblocking, it was possible to execute the next command before producer was actually closed